README.md

```diff
@@ -111,12 +112,31 @@ location @extensionless-php {
 }
 ```
 
+If you've put your nginx configuration in a file in `sites_available`, link it and test nginx
+
+```sh
+function link_nginx(){
+  domain="$1"
+  sudo ln -s /etc/nginx/sites-available/$domain /etc/nginx/sites-enabled/$domain
+}
+link_nginx your_domain.com
+sudo nginx -t
+```
+
 Start php-fpm and nginx immediately and on every boot:
 
 ```sh
 sudo systemctl enable --now php-fpm nginx
 ```
 
+The name of the php-fpm file might vary depending on the version; you can find it with
+
+```
+sudo systemctl list-unit-files | grep php.*fpm
```

Comment:
sudo is probably not needed on this line, since i believe that listing unit files doesnt require any privileges